pkgs/by-name/te/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv,  pkgs ? import <nixpkgs> {} }:
+
+pkgs.stdenv.mkDerivation {
+  pname = "temp-throttle";
+  version = "3.01";
+  src = pkgs.fetchFromGitHub {
+    owner = "Sepero";
+    repo = "temp-throttle";
+    rev = "fd4a429a3cba0eb980f91c81e9ef5bd70f4d94c2";
```

Comment:
Please create a tag/release in the upstream repo, and refer to it like this:
`rev = "v${version}";`